lib/bitcrowd_ecto/assertions.ex

```diff
@@ -323,18 +323,18 @@ defmodule BitcrowdEcto.Assertions do
       end
   """
   @doc since: "0.1.0"
-  @spec assert_count_difference(Ecto.Repo.t(), module, integer, (() -> any)) ::
+  @spec assert_count_difference(Ecto.Repo.t(), module, integer, (() -> any), keyword) ::
           Changeset.t() | no_return
-  def assert_count_difference(repo, schema, by, how) do
-    assert_difference(fn -> repo.count(schema) end, by, how,
+  def assert_count_difference(repo, schema, by, how, opts \\ []) when is_integer(by) do
```

Comment:
why the `is_integer` guard? Not that it hurts, only wondering because it's now the only guard on this function.

Reply:
I don't 100% remember, but I think this was to enforce a bit that you have the right 5 arguments if passing options and you're not, for example, forgetting `repo` and having this fail later on since adding options will make this seem like the right arity. it made more sense at the time but now seems like unnecessarily confusing.

Happy to remove it. or maybe `is_function(how)` makes a bit more sense

Reply:
removed